docs/Manual/Deployment/Kubernetes/DeploymentReplicationResource.md

```diff
@@ -103,8 +146,62 @@ the TLS connection created by the SyncMaster at the specified destination.
+- If `spec.X.deploymentName` is set, the JWT secret found in the deployment is used.
+
+## Creating client authentication certificate keyfiles
+
+The client authentication certificates needed for the `Secrets` identified by `spec.source.auth.keyfileSecretName` & `spec.destination.auth.keyfileSecretName`
+are normal ArangoDB keyfiles that can be created by the `arangosync create client-auth keyfile` command.
+In order to do so, you must have access to the client authentication CA of the source/destination.
+
+If the client authentication CA at the source/destination also contains a private key (`ca.key`), the ArangoDeployment operator
+can be used to create such a keyfile for you, without the need to have `arangosync` installed locally.
+Read the following paragraphs for instructions on how to do that.
+
+## Creating and using access packages
+
+An access package is a YAML file that contains:
+
+- A client authentication certificate, wrapped in a `Secret` in a `tls.keyfile` data field.
+- A TLS certificate authority public key, wrapped in a `Secret` in a `ca.crt` data field.
+
+The format of the access package is such that it can be inserted into a Kubernetes cluster using the standard `kubectl` tool.
+
+To create an access package that can be used to authenticate with the ArangoDB SyncMasters of an `ArangoDeployment`,
+add a name of a non-existing `Secret` to the `spec.sync.externalAccess.accessPackageSecretNames` field of the `ArangoDeployment`.
+In response, a `Secret` is created in that Kubernetes cluster, with the given name, that contains a `accessPackage.yaml` data field
+that contains a Kubernetes resource specification that can be inserted into the other Kubernetes cluster.
+
+The process for creating and using an access package to authentication at the source cluster is as follows:
+
+- Edit the `ArangoDeployment` resource of the source cluster, set `spec.sync.externalAccess.accessPackageSecretNames` to `["my-access-package"]`
+- Wait for the `ArangoDeployment` operator to create a `Secret` named `my-access-package`.
+- Extract the access package from the Kubernetes source cluster using:
+
+```bash
+kubectl get secret my-access-package --template='{{index .data "accessPackage.yaml"}}' | base64 -D > accessPackage.yaml
+```
+
+- Insert the secrets found in the access package in the Kubernetes destination cluster using:
+
+```bash
+kubectl apply -f accessPackage.yaml
+```
 
-This setting specifies the name of a `Secret` containing a client authentication certificate,
-used to authenticate the SyncMaster in the destination cluster with the SyncMaster in the
-source cluster.
+As a result, the destination Kubernetes cluster will have 2 additional `Secrets`. One containing a client authentication certificate
+formatted ad a keyfile. Another containing the public key of the TLS CA certificate of the source cluster.
```

Comment:
spelling or grammar or typo:  "One containing a client authentication certificate formatted ad a keyfile."  ad -> and ?  If so, better wording would be "One contains a ...".  Second sentence would then be "Another contains ..."